datacube/storage/storage.py

```diff
@@ -61,7 +61,10 @@ def _rasterio_resampling_method(resampling):
 
 if str(rasterio.__version__) >= '0.36.0':
     def _rasterio_crs_wkt(src):
-        return str(src.crs.wkt)
+        if src.crs:
+            return str(src.crs.wkt)
```

Comment:
In the new rasterio version, `src.crs` will be `None`, but `CRS().wkt` raises an exception.
In the old version, `CRS().wkt` just returned an empty string?